app/command_mute.go

```diff
@@ -43,14 +43,20 @@ func (me *MuteProvider) DoCommand(a *App, args *model.CommandArgs, message strin
 		return &model.CommandResponse{Text: args.T("api.command_mute.error", map[string]interface{}{"Channel": channel.DisplayName}), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
 	}
 
+	channelName := ""
+	splitMessage := strings.Split(message, " ")
 	// Overwrite channel with channel-handle if set
 	if strings.HasPrefix(message, "~") {
```

Comment:
can use this? or not `channelName := strings.TrimPrefix(message, "~")`

Reply:
this won't work cause I want to avoid things like `~channelName and more stuff`

Reply:
ok, thanks!